deeprankcore/dataset.py

```diff
@@ -52,24 +52,203 @@ def save_hdf5_keys(
                 f_dest[key] = h5py.ExternalLink(f_src_path, "/" + key)
 
 
-class GraphDataset(Dataset):
+class DatasetParent(Dataset):
+
+    def __init__(self, # pylint: disable=too-many-arguments
+                 hdf5_paths: Union[str, List[str]],
```

Comment:
Maybe something like `db_path` (database path, at singular) would be better? Just because it's a parameter really visible to the user, otherwise I wouldn't care

Reply:
in 123e5a0 I named it hdf5_path on the outside, but internally it will be called hdf5_paths.